tests/controllers/userController.py

```diff
@@ -0,0 +1,159 @@
+import cherrypy
+
+# from services.userServiceProvider import UserServiceProvider
+
+from typing import Dict, List
```

Comment:
This won't work for us, since cherrypy supports Python 2.7–3.6.
In #1510 I explained, why using native typing syntax is not an option for the project. You might want use comment-style typings instead.